man/systemd-sleep.conf.xml

```diff
@@ -173,11 +175,10 @@
       </varlistentry>
       <varlistentry>
         <term><varname>HibernateDelaySec=</varname></term>
-
-        <listitem><para>The amount of time the system spends in suspend mode before the system is
-        automatically put into hibernate mode, when using
+        <listitem><para>The amount of time the system spends in suspend mode before the RTC alarm wakes the system
```

Comment:
`The amount of time the system spends in suspend mode before the RTC alarm wakes the system, before the battery discharge rate can be estimated and used instead to calculate the suspension interval.`